### ISSUE TITLE
Require user to give email address for NCBI service access

### ISSUE THREAD

**Issue by renzok** [StuntsPT/NCBI_Mass_Downloader#13]
NCBI requires that an email is provided in order to allow contacting the user in case of issues.

However, currently you provide your email. This is not a solution, because the user of your software will never receive an email from NCBI in case issues occured and second it is a privacy issue that you will receive the email instead. As a user, I do not want that some one else (in this case) gets to know how I use your software without my explicit permission.

**Comment by StuntsPT** [comment]
Yes, that was my initial thought too. The user would provide NCBI with their email address, so that in case of "abuse" they could contact them.
The way I interpreted [Biopython documentation]()https://github.com/biopython/biopython/blob/master/Bio/Entrez/__init__.py#L525-L532 also let me to believe this.

However, as I was starting to build the FAQ I decided that the email field explanation should be referencing instead a more "official" source, and thus, I dug the [NCBI documentation](http://www.ncbi.nlm.nih.gov/books/NBK25497/#_chapter2_Usage_Guidelines_and_Requiremen_) for this.

What is stated there is the the tool should provide it's name and the developer's email. 
I have registered *NCBI Mass Sequence Downloader* with them already.

I suppose the rationale behind this is that they want to make sure that the tools people are using do not abuse the API (eg. they respect the rule of no more than 3 requests per second). If that is so, I suppose it must me more usefull to contact the tool devs than their users...

Nevertheless, I prefer to follow the official documentation than the hints in biopython. As such, I have removed the email field from both the GUI and the CLI interfaces, and expunged it from the documentation.
